docusaurus/docs/Android/04-compose/04-message-components/02-message-list-header.mdx

```diff
@@ -12,34 +18,35 @@ To use the component, simply combine it with the rest of your UI, e.g. in `setCo
 ```kotlin
 override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    
+
     // Load the data for the header here
-    
+
     setContent {
-        Column(Modifier.fillMaxSize()) {
-            MessageListHeader(
-                channel,
-                currentUser,
-                connectionState,
-                messageMode,
-                modifier = Modifier.fillMaxWidth(),
-                onBackPressed = { },
-                onHeaderActionClick = { },
-            )
-
-            // Rest of your UI
+        ChatTheme {
+            Column(Modifier.fillMaxSize()) {
+                MessageListHeader(
+                    modifier = Modifier.wrapContentHeight(),
+                    channel = channel,
+                    currentUser = currentUser,
+                    connectionState = connectionState,
+                    messageMode = messageMode,
+                    onBackPressed = { },
+                    onHeaderActionClick = { },
+                )
+
+                // Rest of your UI
+            }
         }
     }
-}
```

Comment:
I think we're missing these braces now - we should add the mback (those should be in line with the onCreate indentation (0))